### PR TITLE
remove deprecated AsmConfig and use new config

### PR DESCRIPTION
**Pull request by niehuayang**
JIRA: https://jira.tools.sap/browse/CXSPA-1449

### CODE REVIEW

**Review by justinlee01 — DISMISSED** [review]

**Review by ercultimate — CHANGES_REQUESTED** [review]
Code is all good.  We just need to document the API changes in the migration documents.

**Comment by niehuayang** [comment]
> Code is all good. We just need to document the API changes in the migration documents.

The API changes have been added in the migration documents, could you please help me review it?

File: docs/migration/6_0/migration-comments-api-elements.json [hunk 1]
```diff
@@ -8,5 +8,10 @@
     "apiElementName": "AwesomeClass",
     "entryPoint": "@spartacus/feature/components",
     "migrationComment": "This class is not needed anymore.  Things are done in this way instead.  (note: this example won't appear in the migration material because the AwesomeClass class does not exist)"
+  },
+  {
+    "apiElementName": "AsmConfig",
+    "entryPoint": "@spartacus/asm/core",
+    "migrationComment": "Use class @spartacus/asm/root/AsmConfig instead."
```
Comment (ercultimate) [inline docs/migration/6_0/migration-comments-api-elements.json]:
```suggestion
    "migrationComment": "Use class 'AsmConfig' from '@spartacus/asm/root' instead."
```

**Review by ercultimate — CHANGES_REQUESTED** [review]
I suggested a clearer migration comment for the documentation.  With that, I think this should be ready for merge.

File: docs/migration/6_0/migration-comments-api-elements.json (hunk 1 above)
Comment (niehuayang) [reply docs/migration/6_0/migration-comments-api-elements.json]: Yea, have changed

**Review by ercultimate — APPROVED** [review]

**Pull request merged**